platform/core/poly-rdf/src/rdf_failure.rs

```diff
@@ -1,8 +1,6 @@
-use oxigraph::{
-    sparql::{EvaluationError, ParseError},
-    store::StorageError,
-};
+use oxigraph::{sparql::EvaluationError, store::StorageError};
 use serde::Serialize;
+use spargebra::ParseError;
```

Comment:
What's the different between this ParseError and the other?

Reply:
EvaluationError -> Something went wrong when executing the query
ParseError -> Query is wrong
StorageError -> yeah